src/test_report.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2022 Jeny Sadadia
+# Author: Jeny Sadadia <jeny.sadadia@gmail.com>
+
+import os
+import sys
+
+import kernelci.config
+import kernelci.data
+from kernelci.cli import Args, Command, parse_opts
+import jinja2
+
+
+class TestReport:
+
+    def __init__(self, configs, args):
+        self._db_config = configs['db_configs'][args.db_config]
+        api_token = os.getenv('API_TOKEN')
+        self._db = kernelci.data.get_db(self._db_config, api_token)
+
+    def run(self):
+        sub_id = self._db.subscribe('node')
+        self._print("Listening for test completion events")
+        self._print("Press Ctrl-C to stop.")
+
+        try:
+            while True:
+                sys.stdout.flush()
+                event = self._db.get_event(sub_id)
+
+                node = self._db.get_node_from_event(event)
+                if node['status'] is None:
+                    continue
+
+                self._print(f"Node: {node}")
```

Comment:
Please remove this now the client as working, as this a debug hack useful only during development.

Reply:
Yes, it was a debug print. Removed the line.